common/es6/index.js

```diff
@@ -0,0 +1,193 @@
+    if (srcSplit.length > 1) {
+      image = srcSplit[1].split('" alt="')[0]
+    }
+    markupCleaned = markupFull.replace(/<img[^>]*>/, '')
+    markupCleaned = markupCleaned.replace(/<img/g, '<img class="img-fluid"')
+  }
+
+  return {
+    urlSlug: urlSlug,
+    url: url,
+    title: title,
+    preview: preview,
+    description: description,
+    image: image,
+    markupFull: markupFull,
+    markup: markupCleaned,
+    blockstackID: blockstackID,
+    date: dateString,
+    datetime: datetimeString
+  }
+}
+
+const getAllPostsFromRSS = (rssXML) => {
+  let posts = []
+  let postObject = {}
+
+  parseString(rssXML, (err, result) => {
+    // parse XML string
+    const firstChannel = result.rss.channel[0]
+    const channelItems = firstChannel.item
+
+    channelItems.map((rssPost) => {
+      let post = getPostFromRSS(rssPost)
+      if (blogAuthors.hasOwnProperty(post.blockstackID)) {
+        post.creator = blogAuthors[post.blockstackID]
+      } else {
+        post.creator = blogAuthors['blockstack.id']
+      }
+      if (post.markup && post.markup.includes('http://mun')) {
+        post.markup = post.markup.replace('http://mun', 'https://mun')
+      }
+      if (post.markupFull && post.markupFull.includes('http://mun')) {
+        post.markupFull = post.markupFull.replace('http://mun', 'https://mun')
+      }
+      posts.push(post)
+      postObject[post.urlSlug] = post
+    })
+  })
+
+  return {
+    posts,
+    postObject
+  }
+}
+
+const filterByDate = (a, b) => a.createdAt < b.createdAt
+function slugify(text) {
+  return text
+    .toString()
+    .toLowerCase()
+    .replace(/\s+/g, '-') // Replace spaces with -
+    .replace(/[^\w\-]+/g, '') // Remove all non-word chars
+    .replace(/\-\-+/g, '-') // Replace multiple - with single -
+    .replace(/^-+/, '') // Trim - from start of text
+    .replace(/-+$/, '') // Trim - from end of text
+}
+
+const fetchJobs = async () => {
+  try {
+    const res = await fetch(
+      'https://api.lever.co/v0/postings/blockstack?mode=json'
+    )
+    if (res.status >= 400) {
+      console.log('Bad response from server')
+    }
+    const jobsArr = await res.json()
+    const DontSeeWhatYoureLookingForID = '5deebd50-43d3-45d1-afa2-55ccab0e812a'
+    const lastItem = jobsArr.find(
+      (listing) => listing.id === DontSeeWhatYoureLookingForID
+    )
+    const sortedByDateJobs = jobsArr
+      .filter((listing) => listing.id !== DontSeeWhatYoureLookingForID)
+      .sort(filterByDate)
+    return [
+      ...sortedByDateJobs,
+      {
+        ...lastItem
+      }
+    ]
+  } catch (error) {
+    console.log('error', error)
+  }
+}
+const fetchStats = async () => {
+  try {
+    const res = await fetch(
+      'https://blockstack-site-api.herokuapp.com/v1/stats'
+    )
+    if (res.status >= 400) {
+      console.log('Bad response from server')
+    }
+    return res.json()
+  } catch (error) {
+    console.log('error', error)
+  }
+}
+
+const urls = [
```

Comment:
The API repo mostly does a proxy to our Ghost RSS feed, and from some googling, it doesn't seem that there is an HTTP parameter to change the limit of how many items you get from the feed. I think we should just iterate through in this code, especially because we're just running this as a pre-build step. I made a separate issue in #658, so we can track it there, and get this out the door.